NMS-14692: Allow destination paths to be triggered from the shell and UI (#6045)
OpenNMS/opennms: docs/modules/operation/pages/deep-dive/notifications/shell.adoc

= Notification Shell Commands

> The following shell commands are made available to help administer and debug Notifd.

Reviewer: ```suggestion
The following shell commands can help you to administer and debug Notifd.
```

== opennms:destination-path-trigger

Use the `destination-path-trigger` command to test the notification targets and commands associated with a destination path.

> This can be used to verify that commands are properly configured and that notifications get sent to the recipients.

Reviewer: ```suggestion
Use to verify that commands are properly configured and that notifications get sent to the recipients.
```
